lambda_layers/core_layer/python/core_layer/handler/comment_handler.py

```diff
@@ -0,0 +1,49 @@
+from uuid import uuid4
+import logging
+from core_layer.db_handler import Session
+from core_layer.model.comment_model import Comment
+
+
+def create_comment(comment, user_id, parent_type, parent_id, timestamp = None, status = None):
+    """
+    Creates comment
+
+    Args:
+        mandatory:
+            comment (str)
+            user_id (str)
+            parent_type (str) needs to be one of: 'item', 'submission', 'review_answer', 'comment'
+            parent_id (str) 
+            session (Session)
+        optional:
+            is_test (bool)
+            timestamp (DateTime)
+            status (str)
+    """
+
+  
+    session = Session()
+
+    comment_obj = Comment()
+
+    comment_obj.id = str(uuid4())
+    comment_obj.comment = comment
+    comment_obj.user_id = user_id
+    if timestamp:
+        comment_obj.timestamp = timestamp
+    if status:
+        comment_obj.status = status
```

Comment:
Are these checks really necessary?  Don't the "default/server_default" settings deal with None values?

Reply:
I think the checks are there to check whether `timestamp`or `status` are defined in the function call. In this case the default would be overwritten?

Reply:
Got that, however if you remove the check and the argument is `None`, then `comment_obj.*` would be initialized with `None` instead of being undefined as it currently is. I would assume the defaults defined in the model would then still be used.

But this is just an assumption and I don't know if this is exactly how SQLAlchemy works.

Reply:
I tried it out and your assumption is right